src/android/io/sqlc/SQLitePlugin.java

```diff
@@ -337,6 +337,11 @@ private static String getDBConnectionName(String dbname, JSONObject options) {
 
             this.q = new LinkedBlockingQueue<DBQuery>();
             this.openCbc = cbc;
+
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
```

Comment:
hm, using this as such won't make the plugin fail to compile in older versions of mabs, due to not using the SDK 30, thus not knowing the value of Build.VERSION_CODES.R ?

Just a heads up, that this is not "retro compatible", therefore almost renders the if useless.

Reply:
Yeah, that's true, and I wondered if we should protected this with the targetSdk too. However, we already have some changes that are not retro compatible with other MABS versions than MABS 7.